repos/fdbt-site/src/pages/manageProductGroup.tsx

```diff
@@ -0,0 +1,510 @@
+                                            {otherProducts.length > 0 ? (
+                                                <div>
+                                                    <h2 className="govuk-heading-m govuk-!-margin-bottom-6">
+                                                        Flat fare products
+                                                    </h2>
+
+                                                    {otherProducts.length > 0 ? (
+                                                        buildOtherProductSection(
+                                                            indexCounter + otherProducts.length,
+                                                            productsSelected,
+                                                            setProductsSelected,
+                                                            otherProducts,
+                                                        )
+                                                    ) : (
+                                                        <p className="govuk-body-m govuk-!-margin-top-5">
+                                                            <em>
+                                                                You currently have no flat fare products that can be
+                                                                added to product group.
+                                                            </em>
+                                                        </p>
+                                                    )}
+                                                </div>
+                                            ) : null}
+                                        </div>
+                                    </div>
+                                </div>
+                            )}
+
+                            <button
+                                type="submit"
+                                className={`govuk-button${
+                                    productsToDisplay.length === 0 ? ' govuk-visually-hidden' : ''
+                                }`}
+                                id="continue-button"
+                            >
+                                Create Product Group
+                            </button>
+                        </div>
+                    </div>
+                </CsrfForm>
+            </BaseLayout>
+        </>
+    );
+};
+
+export const getServerSideProps = async (
+    ctx: NextPageContextWithSession,
+): Promise<{ props: ManageProductGroupProps }> => {
+    const noc = getAndValidateNoc(ctx);
+    const products = await getAllProductsByNoc(noc);
+    const nonExpiredProducts = getNonExpiredProducts(products);
+    const nonExpiredProductsWithActiveServices = await filterOutProductsWithNoActiveServices(noc, nonExpiredProducts);
+    const allPassengerTypes = await getAllPassengerTypesByNoc(noc);
+
+    const allProductsToDisplay: ProductToExport[] = await Promise.all(
```

Comment:
Can we rename the type 'ProductToExport' to something else, so it makes more sense here?

Reply:
if we need to change ... what about the ProductToDisplay? 

this will trigger change in the exports also

Reply:
Yeah just rename all instances of ProductToExport to ProductToDisplay